plugins/woocommerce/client/admin/client/settings-payments/components/offline-gateway-list-item/offline-payment-gateway-list-item.tsx

```diff
@@ -69,7 +79,7 @@ export const OfflinePaymentGatewayListItem = ( {
 									gateway.onboarding._links.onboard.href
 								}
 								isOffline={ true }
-								gatewayHasRecommendedPaymentMethods={ false }
+								gatewayHasRecommendedPaymentMethods={ false } // offline gateway items don't have recommended PMs.
```

Comment:
Start with uppercase.

Reply:
thanks! addressed in https://github.com/woocommerce/woocommerce/pull/54263/commits/410a08d7ccf0886fc7e9b53f3cc99ef1a8d78218